### PR TITLE
Add seed groups to migrate task

### PR DESCRIPTION
**Pull request by mjgiarlo**
connects to LD4P/sinopia#196

Also includes:

* Store group slugs and labels in default configuration.
* Convert relevant TTL fixtures to Mustache templates to allow injection of values.
* Wrap migration functionality in a `bin/` script, rather than an npm script. This allows us to export the migrate task to dependents.
* When migrating, sniff whether resources exist before creating them. This tells us whether we should use the POST verb or the PATCH verb, and allows us to safely run the migrate task on a fully populated repository.
* Use Cognito token in population script in case running against populated repository.
* Sleep between migration tasks to give Trellis a chance to persist data.

### CODE REVIEW

File: fixtureWAC/repositoryContainer.ttl [hunk 1]
```diff
@@ -2,5 +2,4 @@ PREFIX ldp: <http://www.w3.org/ns/ldp#>
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 
 <>
-        a ldp:BasicContainer, ldp:Container ;
```
Comment (ndushay) [inline fixtureWAC/repositoryContainer.ttl]: gosh, my empirical experience was I absolutely needed BasicContainer specified or it didn't do the right thing --  the repositoryContainer ttl somehow landed in the root container, and there was no parent to child container relationship.

Have you double checked it does what you want by checking the root container afterwards?  (there prolly should be a test for that)

Comment (mjgiarlo) [reply fixtureWAC/repositoryContainer.ttl]: @ndushay I have, yeah. I've run it 5-10 times and it results in the right structure every time. Trellis ignores rdf type assertions in the payload.

Comment (mjgiarlo) [reply fixtureWAC/repositoryContainer.ttl]: See the root container (w/ prefixes removed for clarity/brevity):

```shell
$ curl http://localhost:8080/

<http://platform:8080/>
        rdfs:label    "Sinopia LDP Root Container" ;
        ldp:contains  <http://platform:8080/repository> .

```

And here's the repository container:

```shell
$ curl http://localhost:8080/repository

<http://platform:8080/repository>
        rdfs:label    "Sinopia Repository Container" ;
        ldp:contains  <http://platform:8080/repository/alberta> ;
        ldp:contains  <http://platform:8080/repository/boulder> ;
        ldp:contains  <http://platform:8080/repository/chicago> ;
        ldp:contains  <http://platform:8080/repository/cornell> ;
        ldp:contains  <http://platform:8080/repository/dlc> ;
        ldp:contains  <http://platform:8080/repository/duke> ;
        ldp:contains  <http://platform:8080/repository/frick> ;
        ldp:contains  <http://platform:8080/repository/harvard> ;
        ldp:contains  <http://platform:8080/repository/hrc> ;
        ldp:contains  <http://platform:8080/repository/ld4p> ;
        ldp:contains  <http://platform:8080/repository/michigan> ;
        ldp:contains  <http://platform:8080/repository/minnesota> ;
        ldp:contains  <http://platform:8080/repository/nlm> ;
        ldp:contains  <http://platform:8080/repository/northwestern> ;
        ldp:contains  <http://platform:8080/repository/pcc> ;
        ldp:contains  <http://platform:8080/repository/penn> ;
        ldp:contains  <http://platform:8080/repository/princeton> ;
        ldp:contains  <http://platform:8080/repository/stanford> ;
        ldp:contains  <http://platform:8080/repository/tamu> ;
        ldp:contains  <http://platform:8080/repository/ucdavis> ;
        ldp:contains  <http://platform:8080/repository/ucsd> ;
        ldp:contains  <http://platform:8080/repository/washington> ;
        ldp:contains  <http://platform:8080/repository/yale> .
```

And a sample group container:

```shell
$ curl http://localhost:8080/repository/stanford

<http://platform:8080/repository/stanford>
        rdfs:label  "Sinopia Stanford University Group Container" .
```

Comment (mjgiarlo) [reply fixtureWAC/repositoryContainer.ttl]: @ndushay Given :point_up:, could you review again when you get a moment?

**Pull request merged**